test/linalg/lu.jl

```diff
@@ -29,8 +29,12 @@ dreal = randn(n)/2
 dimg  = randn(n)/2
 
 for eltya in (Float32, Float64, Complex64, Complex128, BigFloat, Int)
-    a = eltya == Int ? rand(1:7, n, n) : convert(Matrix{eltya}, eltya <: Complex ? complex.(areal, aimg) : areal)
-    d = eltya == Int ? Tridiagonal(rand(1:7, n-1), rand(1:7, n), rand(1:7, n-1)) : convert(Tridiagonal{eltya}, eltya <: Complex ? Tridiagonal(complex.(dlreal, dlimg), complex.(dreal, dimg), complex.(dureal, duimg)) : Tridiagonal(dlreal, dreal, dureal))
+    a = eltya == Int ? rand(1:7, n, n) :
+        convert(Matrix{eltya}, eltya <: Complex ? complex.(areal, aimg) : areal)
+    d = eltya == Int ? Tridiagonal(rand(1:7, n-1), rand(1:7, n), rand(1:7, n-1)) :
+        convert(Tridiagonal{eltya}, eltya <: Complex ?
+        Tridiagonal(complex.(dlreal, dlimg), complex.(dreal, dimg), complex.(dureal, duimg)) :
+        Tridiagonal(dlreal, dreal, dureal))
```

Comment:
Perhaps indent the two `Tridiagonal(...` lines, making their nesting within the `convert` more obvious?